lib/srv/desktop/rdp/rdpclient/client.go

```diff
@@ -362,6 +366,8 @@ func handle_bitmap(handle C.uintptr_t, cb C.CGOBitmap) C.CGOError {
 }
 
 func (c *Client) handleBitmap(cb C.CGOBitmap) C.CGOError {
+	c.UpdateClientActivity()
```

Comment:
This shouldn't be here. "Client Activity" is input (mouse move, keyboard, scroll wheel, etc)